src/picknik_ur_base_config/objectives/interpolate_to_joint_state.xml

```diff
@@ -2,7 +2,11 @@
 <root BTCPP_format="4" main_tree_to_execute="Interpolate to Joint State">
     <BehaviorTree ID="Interpolate to Joint State" _description="Move to a specified joint state">
         <Control ID="Sequence" name="root">
-            <SubTree ID="InterpolateToJointStateWaypoint" target_joint_state="{interpolate_to_joint_state.target_joint_state}"/>
+            <Fallback name="wait_for_approval_if_user_available">
+            <Action ID="GetJointStateFromUser" parameter_name="interpolate_to_joint_state.target_joint_state" parameter_value="{target_joint_state}" />
+                <Action ID="LogMessage" message="This objective is used by joint sliders in the Joints tab." log_level="error" />
+            </Fallback>
```

Comment:
A few things to mention here:

- Fix the indentation for the XML.
- Write the `Fallback` node in the same way  as the objective editor in the UI serializes it, which is `Control ID="Fallback" name =...`.
- The fallback condition needs to be a `Sequence` where the first child is `LogMessage` and the second child is `AlwaysFailure` (or, put `LogMessage` under a `ForceFailure` decorator, but I find that using `AlwaysFailure` is more readable because it shows the steps clearly). What happens right now when we run this objective as currently written is that `GetJointStateFromUser` runs and fails because no user input was provided, which triggers the fallback branch. Since `LogMessage` succeeds when it runs, it makes the fallback control succeed, so the objective keeps running and the subtree that does the motion planning and execution ends up being run even though it should not.